spec/steps/provision.fmf

```diff
@@ -68,3 +68,20 @@ example: |
             how: beaker
             family: Fedora31
             tag: released
+
+/connect:
+    summary: Connect to a provisioned box
+    description:
+        Do not provision a new system. Instead, use provided
+        authentication data to connect to a running machine.
+    example: |
+        provision:
+            how: connect
+            guest: hostname or ip address
+            user: username
+            password: password
+
+        provision:
+            how: connect
+            guest: hostname or ip address
+            key: private-key
```

Comment:
```private-key-file``` would be better

Reply:
Well, that's where aliases come in IMHO :).

Reply:
I think it would be good to keep the command line options consistent with the attribute names in L2 metadata. So if you both suggest that `key` is not enough specifi, I'd propose to use `private-key` in both cli and metadata. Unless really needed I would omit the `file` suffix, this can be included in the description.

Reply:
@thrix, or did you mean `s/private-key/private-key-path/` instead?

Reply:
Updated the definiton to use `private-key-path` for both spec and cli story.

Reply:
private-key-path is better :)

Reply:
Ok, updated to `private-key-path`.